env0/utils.go

```diff
@@ -489,3 +489,12 @@ func ttlToDuration(ttl *string) (time.Duration, error) {
 
 	return time.ParseDuration(fmt.Sprintf("%dh", hours))
 }
+
+func lastSplit(s string, sep string) []string {
+	lastIndex := strings.LastIndex(s, sep)
+	if lastIndex == -1 {
+		return nil
```

Comment:
why `nil`? the original split would return the original string, no? that might confuse when using that method.
in the case of this fix, the outcome will remain that the length of the desired array won't be 2, so IMO it's better to stick to known outcomes

Reply:
fair point. I'll change that.

Reply:
fixed.